### PR TITLE
Updates in line with wider schema changes

### PR DESCRIPTION
**Pull request by IWHarvey**

### CODE REVIEW

File: local_land_charges_api_stub/documentation/local-land-charges-api.yml [hunk 1]
```diff
@@ -628,62 +664,82 @@ definitions:
...
   tribunal-definitive-certificate-date:
-    description: "The date the definitive Lands Tribunal certificate was issued by the Lands Tribunal. Valid only when 'charge-type' is 'Light Obstruction Notice' and documents-filed includes a definitive-land-certificate"
+    description: The date the definitive Lands Tribunal certificate was issued by the Lands Tribunal. Valid only when 'charge-type' is 'Light Obstruction Notice'
     type: string
     pattern: '^\d{4}-\d{2}-\d{2}$'
     format: date
     example: '2017-01-04'
   documents-filed:
-    description: "The documents lodged to support the LON registration. Valid only when 'charge-type' is 'Light Obstruction Notice'"
-    enum: ["form-a", "colour-plan", "definitive-certificate"]
```
Comment (Griff505) [inline local_land_charges_api_stub/documentation/local-land-charges-api.yml]: This should be left as is. LONs can't be added or amended via the api, only viewed, and only the descriptions of the available documents are shown not the actual locations. However it needs temporary-lon-certificate added?

File: local_land_charges_api_stub/documentation/local-land-charges-api.yml [hunk 2]
```diff
@@ -628,62 +664,82 @@ definitions:
     description: 'The name of the Minister of the Crown, government department, local authority or other person by whom the charge is brought into existence'
     type: string
     minLength: 1
-    example: 'Exeter City Council'
+    example: 'City of Exeter'
```
Comment (Griff505) [inline local_land_charges_api_stub/documentation/local-land-charges-api.yml]: This should be left as is, it was in correct format.

File: local_land_charges_api_stub/documentation/local-land-charges-api.yml [hunk 3]
```diff
@@ -54,8 +54,14 @@ paths:
...
                 charge-type:
                   $ref: '#/definitions/charge-type'
+                charge-sub-category:
+                  $ref: '#/definitions/charge-sub-category'
                 charge-geographic-description:
                   $ref: '#/definitions/charge-geographic-description'
+                charge-addresss:
+                  $ref: '#/definitions/charge-address'
+                registration-date:
+                  $ref: '#/definitions/registration-date'
```
Comment (Griff505) [inline local_land_charges_api_stub/documentation/local-land-charges-api.yml]: We don't allow registration date to be specified on 'add charge', it gets auto filled.

File: local_land_charges_api_stub/documentation/local-land-charges-api.yml (hunk 1 above)
Comment (IWHarvey) [reply local_land_charges_api_stub/documentation/local-land-charges-api.yml]: This has been changed to a $ref: to the actual object (the enum's not correct)

File: local_land_charges_api_stub/documentation/local-land-charges-api.yml (hunk 2 above)
Comment (IWHarvey) [reply local_land_charges_api_stub/documentation/local-land-charges-api.yml]: Restored. No idea what happened

**Comment by IWHarvey** [comment]
@Griff505 I've removed expiry-text and restored some of the examples I broke (no idea how... actually, I do - I copied the swagger spec from LLC-API)

File: local_land_charges_api_stub/documentation/local-land-charges-api.yml (hunk 1 above)
Comment (IWHarvey) [reply local_land_charges_api_stub/documentation/local-land-charges-api.yml]: Per discussion, fixed

**Pull request merged**